src/test/java/com/example/demo/services/BooksServiceImplTest.java

```diff
@@ -16,16 +16,17 @@
 import java.util.ArrayList;
 import java.util.List;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.*;
 
 @RunWith(MockitoJUnitRunner.class)
 @SpringBootTest
 public class BooksServiceImplTest {
     @Mock
     private BookRepository bookRepository;
+    @Mock
+    private BookMainInfo bookMainInfo;
```

Comment:
To nie może byc zmokowane. To jest nasz input, definiujemy to w kazdym teście i oczekujemy jakiś wyników

Reply:
done